.gitignore

```diff
@@ -1,3 +1,4 @@
-.vscode/
-node_modules/
+node_modules
 .idea/
+.yarn/
+.DC_Store
```

Comment:
Wrong file name

Reply:
Also this file should be removed from git. Currently it is included to PR as a newly added file